components/src/components/shared/pricing-tiers-tooltip/PricingTiersTooltip.tsx

```diff
@@ -103,7 +104,9 @@ export const PricingTiersTooltip = ({
               );
             })}
           </dl>
-          {tiersMode && (
+
+          {(tiersMode === TiersMode.Volume ||
```

Comment:
Is this meant to check for non-null/non-undefined tiersMode? or we do specifically care about these two modes?